src/Service/EntryCommentManager.php

```diff
@@ -159,7 +159,7 @@ public function purge(User $user, EntryComment $comment): void
         $this->dispatcher->dispatch(new EntryCommentBeforePurgeEvent($comment, $user));
 
         $magazine = $comment->entry->magazine;
-        $image = $comment->image?->filePath;
+        $image = $comment->image?->getId();
```

Comment:
`image?` doesn't that mean that `image` valuable can be empty? Meaning `getId()` will cause errors in some cases (null error).

Reply:
`$image?->...` is the same as `image?.something` in c# for example. Meaning it will only call the `something` part if `image` is not null. https://wiki.php.net/rfc/nullsafe_operator. `image` can be `null`, but it is checked for that before it is used

Reply:
Yes ok. we are safe here.